proto/firecracker.proto

```diff
@@ -102,5 +102,11 @@ message JailerConfig {
     string Mems = 3;
     uint32 UID = 4;
     uint32 GID = 5;
+	// CgroupPath is used to dictate where the cgroup should be located
+	// relative to the cgroup directory which is
+	// /sys/fs/cgroup/cpu/<CgroupPath>/<vmID>
```

Comment:
@ofiliz Now clients can inject an arbitrary string, but still cannot remove the last vmID part. Does it work?

Reply:
That's the same VMID in the CreateVMRequest, right? Yes, that should be fine.